docs/en/sql-reference/statements/alter/projection.md

```diff
@@ -0,0 +1,23 @@
+---
+toc_priority: 49
+toc_title: PROJECTION
+---
+
+# Manipulating Projections {#manipulations-with-projections}
+
+The following operations are available:
+
+-   `ALTER TABLE [db].name ADD PROJECTION name AS SELECT <COLUMN LIST EXPR> [WHERE] [GROUP BY] [ORDER BY]` - Adds projection description to tables metadata.
+
+-   `ALTER TABLE [db].name DROP PROJECTION name` - Removes projection description from tables metadata and deletes projection files from disk.
+
+-   `ALTER TABLE [db.]table MATERIALIZE PROJECTION name IN PARTITION partition_name` - The query rebuilds the projection `name` in the partition `partition_name`. Implemented as a [mutation](../../../../sql-reference/statements/alter/index.md#mutations).
+
+-   `ALTER TABLE [db.]table CLEAR PROJECTION name IN PARTITION partition_name`
```

Comment:
-   `ALTER TABLE [db.]table CLEAR PROJECTION name [IN PARTITION partition_name]` - Deletes projection files from disk without removing description.